core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -579,8 +607,8 @@ void checkTieredStorage() {
     int globalTiers = getInt(PropertyKey.MASTER_TIERED_STORE_GLOBAL_LEVELS);
     Set<String> globalTierAliasSet = new HashSet<>();
     for (int i = 0; i < globalTiers; i++) {
-      globalTierAliasSet
-          .add(get(PropertyKey.Template.MASTER_TIERED_STORE_GLOBAL_LEVEL_ALIAS.format(i)));
+      globalTierAliasSet.add(
+              getString(PropertyKey.Template.MASTER_TIERED_STORE_GLOBAL_LEVEL_ALIAS.format(i)));
```

Comment:
nit: we follow Google Java style of indentation: 

https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent